openapi.yaml

```diff
@@ -25,254 +37,677 @@ security:
   - OAuth2:
       - full
 paths:
-  /pets:
+  /pizzas:
     get:
-      summary: List all pets
+      description: "Get all pizzas"
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - pizza
       parameters:
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
         - in: query
-          name: species
-          description: Search by pet species
+          name: name
+          description: Search pizzas by name
+          required: false
+          type: string
+        - in: query
+          name: ovenTemp
+          description: Search pizzas by bake temperature (in degrees Fahrenheit)
           required: false
           type: string
+        - in: query
+          name: bakeTime
+          description: Search pizzas by baking time (in minutes)
+          type: string
       responses:
-        '200':
-          description: Successful response
+        200:
+          description: "A list of pizzas"
           schema:
-            $ref: '#/definitions/PetsResult'
-        '400':
-          $ref: '#/responses/400'
-        '500':
-          $ref: '#/responses/500'
+            $ref: "#/definitions/PizzasResults"
+        500:
+          $ref: "#/responses/500"
     post:
-      summary: Add a new pet
-      operationId: postPet
+      description: "Create a new pizza recipe"
       tags:
-        - pets
+        - pizza
       parameters:
         - in: body
-          name: body
+          name: attributes
+          description: attributes to assign to the new pizza
+          required: true
+          schema:
+            $ref: "#/definitions/PizzaAttributes"
+      responses:
+        200:
```

Comment:
Based on [JSON API - create](https://jsonapi.org/format/#crud-creating-responses-201), if the resource has been successfully created, it should return a `201`.